misc/packaging/release.json

```diff
@@ -1,68 +1,64 @@
 {
-   "//":"Configuration for packaging ACRN and Ubuntu for simple installation",
+	"//":"release ubuntu as sos verion",
+	"install_package":"true",
+	"gcc_version":"7.3.0",
+	"binutils":"2.27",
+	"//":"acrn-hypervisor config",
+	"build_acrn":"true",
+	"sync_acrn_code":"true",
+	"acrn_repo":"https://github.com/projectacrn/acrn-hypervisor.git",
+	"release_version":"remotes/origin/release_2.1",
+	"acrn_deb_package":"true",
+	"install_acrn_deb":"false",
+	"build_cmd":
+	{
+		"build_method":
+		{
+			"use_xml":"true",
+			"use_sourcecode":"true"
+		},
+		"scenario":
+		{
+			"industry":"true",
+			"hybrid":"true",
+			"logical_partition":"true"
 
-   "//":"if true, the script will first install all needed apps and libraries and verify gcc and binutils versions ",
-   "install_package":"true",
-   "gcc_version":"7.3.0",
-   "binutils":"2.27",
+		},
+		"board":
+		{
+			"nuc7i7dnb":"true",
+			"whl-ipc-i5":"true"
+		},
+		"release":"0"
+	},
+	"acrn_patch":
+	{
+		"patch_need":"false",
+		"patch_dir":"acrn_additional_patch",
+		"patch_list":
+		{
+			"0001-xxxxxxxx.patch": "true",
```

Comment:
I guess these are placeholders for patches that don't currently exist?

Reply:
yes, patch should provide by user